spec/system/support_interface/managing_provider_users_spec.rb

```diff
@@ -5,7 +5,6 @@
   include DsiAPIHelper
 
   scenario 'creating a new provider user' do
-    given_the_send_dfe_sign_in_invitations_feature_flag_is_on
     and_dfe_signin_is_configured
```

Comment:
Nit pick, feel free to ignore. 😅 

```suggestion
    given_dfe_signin_is_configured
```

Reply:
Good spot!